assets/queries/terraform/aws/rds_database_cluster_not_encrypted/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "656880aa-1388-488f-a6d4-8f73c23149b2",
+  "queryName": "RDS Database Cluster not Encrypted",
+  "severity": "MEDIUM",
```

Comment:
I think, based on query "**RDS Storage Not Encrypted**", this severity should be `HIGH`, since both queries check if database is encrypted or not... Could you confirm, @JoaoDuarteGomes, please?

Reply:
Agreed. Well spotted @felipe-avelar.